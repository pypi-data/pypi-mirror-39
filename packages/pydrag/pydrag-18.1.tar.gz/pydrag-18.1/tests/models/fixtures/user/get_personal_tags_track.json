{
    "interactions": [
        {
            "request": {
                "body": null,
                "headers": {
                    "Accept": [
                        "*/*"
                    ],
                    "Accept-Encoding": [
                        "gzip, deflate"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "User-Agent": [
                        "python-requests/2.20.0"
                    ]
                },
                "method": "GET",
                "uri": "https://ws.audioscrobbler.com/2.0/?api_key=LAST_FM_API_KEY&format=json&limit=50&method=user.getPersonalTags&page=1&tag=rock&taggingtype=track&user=rj"
            },
            "response": {
                "body": {
                    "string": "{\"taggings\":{\"tracks\":{\"track\":[{\"name\":\"Arc Of A Diver\",\"duration\":\"FIXME\",\"mbid\":\"CENSOREDccc4\",\"url\":\"https://www.last.fm/music/Steve+Winwood/_/Arc+Of+A+Diver\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Steve Winwood\",\"mbid\":\"CENSOREDc68e\",\"url\":\"https://www.last.fm/music/Steve+Winwood\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}]},{\"name\":\"Finish What Ya Started\",\"duration\":\"FIXME\",\"mbid\":\"CENSORED11c8\",\"url\":\"https://www.last.fm/music/Van+Halen/_/Finish+What+Ya+Started\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Van Halen\",\"mbid\":\"CENSORED7c15\",\"url\":\"https://www.last.fm/music/Van+Halen\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}]},{\"name\":\"Cry For Freedom\",\"duration\":\"FIXME\",\"mbid\":\"CENSOREDcc78\",\"url\":\"https://www.last.fm/music/White+Lion/_/Cry+For+Freedom\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"White Lion\",\"mbid\":\"CENSORED44bf\",\"url\":\"https://www.last.fm/music/White+Lion\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}]},{\"name\":\"Good Times Bad Times\",\"duration\":\"FIXME\",\"mbid\":\"CENSORED189f\",\"url\":\"https://www.last.fm/music/Led+Zeppelin/_/Good+Times+Bad+Times\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"Led Zeppelin\",\"mbid\":\"CENSORED674a\",\"url\":\"https://www.last.fm/music/Led+Zeppelin\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}]},{\"name\":\"Hells Bells\",\"duration\":\"FIXME\",\"mbid\":\"CENSOREDb286\",\"url\":\"https://www.last.fm/music/AC%2FDC/_/Hells+Bells\",\"streamable\":{\"#text\":\"0\",\"fulltrack\":\"0\"},\"artist\":{\"name\":\"AC/DC\",\"mbid\":\"CENSORED3ed1\",\"url\":\"https://www.last.fm/music/AC%2FDC\"},\"image\":[{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png\",\"size\":\"small\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png\",\"size\":\"medium\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png\",\"size\":\"large\"},{\"#text\":\"https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png\",\"size\":\"extralarge\"}]}]},\"@attr\":{\"user\":\"RJ\",\"tag\":\"rock\",\"page\":\"1\",\"perPage\":\"50\",\"totalPages\":\"1\",\"total\":\"5\"}}}"
                },
                "headers": {
                    "Access-Control-Allow-Methods": [
                        "POST, GET, OPTIONS"
                    ],
                    "Access-Control-Allow-Origin": [
                        "*"
                    ],
                    "Access-Control-Max-Age": [
                        "86400"
                    ],
                    "Access-Control-Request-Headers": [
                        "Origin, X-Atmosphere-tracking-id, X-Atmosphere-Framework, X-Cache-Date, Content-Type, X-Atmosphere-Transport, *"
                    ],
                    "Connection": [
                        "keep-alive"
                    ],
                    "Content-Length": [
                        "3965"
                    ],
                    "Content-Type": [
                        "application/json; charset=UTF-8"
                    ],
                    "Date": [
                        "Sat, 10 Nov 2018 11:28:08 GMT"
                    ],
                    "Server": [
                        "openresty/1.9.7.5"
                    ]
                },
                "status": {
                    "code": 200,
                    "message": "OK"
                }
            }
        }
    ],
    "version": 1
}
