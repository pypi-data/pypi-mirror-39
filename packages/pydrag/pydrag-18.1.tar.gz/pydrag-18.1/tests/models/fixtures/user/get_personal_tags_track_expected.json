{
    "data": [
        {
            "artist": {
                "mbid": "CENSOREDc68e",
                "name": "Steve Winwood",
                "url": "https://www.last.fm/music/Steve+Winwood"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSOREDccc4",
            "name": "Arc Of A Diver",
            "url": "https://www.last.fm/music/Steve+Winwood/_/Arc+Of+A+Diver"
        },
        {
            "artist": {
                "mbid": "CENSORED7c15",
                "name": "Van Halen",
                "url": "https://www.last.fm/music/Van+Halen"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSORED11c8",
            "name": "Finish What Ya Started",
            "url": "https://www.last.fm/music/Van+Halen/_/Finish+What+Ya+Started"
        },
        {
            "artist": {
                "mbid": "CENSORED44bf",
                "name": "White Lion",
                "url": "https://www.last.fm/music/White+Lion"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSOREDcc78",
            "name": "Cry For Freedom",
            "url": "https://www.last.fm/music/White+Lion/_/Cry+For+Freedom"
        },
        {
            "artist": {
                "mbid": "CENSORED674a",
                "name": "Led Zeppelin",
                "url": "https://www.last.fm/music/Led+Zeppelin"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSORED189f",
            "name": "Good Times Bad Times",
            "url": "https://www.last.fm/music/Led+Zeppelin/_/Good+Times+Bad+Times"
        },
        {
            "artist": {
                "mbid": "CENSORED3ed1",
                "name": "AC/DC",
                "url": "https://www.last.fm/music/AC%2FDC"
            },
            "duration": 0,
            "image": [
                {
                    "size": "small",
                    "text": "https://lastfm-img2.akamaized.net/i/u/34s/CENSORED.png"
                },
                {
                    "size": "medium",
                    "text": "https://lastfm-img2.akamaized.net/i/u/64s/CENSORED.png"
                },
                {
                    "size": "large",
                    "text": "https://lastfm-img2.akamaized.net/i/u/174s/CENSORED.png"
                },
                {
                    "size": "extralarge",
                    "text": "https://lastfm-img2.akamaized.net/i/u/300x300/CENSORED.png"
                }
            ],
            "mbid": "CENSOREDb286",
            "name": "Hells Bells",
            "url": "https://www.last.fm/music/AC%2FDC/_/Hells+Bells"
        }
    ],
    "limit": 50,
    "page": 1,
    "tag": "rock",
    "total": 5,
    "user": "RJ"
}
